osquery/core/database/rocksdb_migration.cpp

```diff
@@ -0,0 +1,281 @@
+/**
+ *  Copyright (c) 2018-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <osquery/core/conversions.h>
+#include <osquery/core/database/rocksdb_migration.h>
+
+#include <boost/filesystem.hpp>
+#include <boost/uuid/uuid.hpp>
+#include <boost/uuid/uuid_generators.hpp>
+#include <boost/uuid/uuid_io.hpp>
+
+namespace osquery {
+
+namespace fs = boost::filesystem;
+
+namespace {
+
+enum DatabaseSchemaVersion {
+  kDatabaseSchemaV1 = 1,
+  kDatabaseSchemaV2 = 2,
```

Comment:
Why do you need such `enum`? The values of it look pretty obvious. Isn't a better way just to define a static const with current version?